app/upgrades/v11/constants.go

```diff
@@ -9,4 +9,11 @@ const (
 	// at the time of this migration, on mainnet, channels 0 to 36 were open
 	// so this migration covers those channels only
 	openChannels = 36
+
+	// FundingAccount is the account which holds the rewards for the incentivized testnet.
+	// It contains ~7.4M tokens, of which ~5.6M are to be distributed.
+	FundingAccount = "evmos1f7vxxvmd544dkkmyxan76t76d39k7j3gr8d45y"
+
+	// The remaining tokens are sent to the Community Pool.
+	CommunityPoolAccount = "evmos1jv65s3grqf6v6jl3dp4t6c9t9rk99cd8974jnh"
```

Comment:
for this, you can just call the `FundCommunityPool` function from the distribution module

Reply:
```suggestion
```